pkg/agent/cache/accessrequest.go

```diff
@@ -56,6 +61,24 @@ func (c *cacheManager) GetAccessRequestByAppAndAPI(appName, remoteAPIID, remoteA
 	return nil
 }
 
+func (c *cacheManager) GetAccessRequestsByApp(appName string) []*v1.ResourceInstance {
```

Comment:
How come this returns the ResourceInstance and not the AccessRequest?

Reply:
It was to keep it consistent with how all the other cache manager functions receive and return ResourceInstance.